packages/applications-service-api/__tests__/unit/utils/document.mapper.test.js

```diff
@@ -0,0 +1,193 @@
+const {
+	mapFilters,
+	mapDocuments,
+	mapBackOfficeDocuments
+} = require('../../../src/utils/document.mapper');
+
+describe('document mapper functions', () => {
+	describe('mapDocuments', () => {
+		it('maps NI documents to v3 response body format', () => {
+			const niDocuments = [
+				{
+					id: 1000,
```

Comment:
Is it worth adding some of the fields that are specific to NI to highlight the differences?
eg,
category
personal_name

Reply:
Good call 👍